src/slider/slider.js

```diff
@@ -83,16 +83,6 @@
   };
 
   /**
-   * Handle mouseup on element.
-   *
-   * @param {Event} event The event that fired.
-   * @private
-   */
-  MaterialSlider.prototype.onMouseUp_ = function(event) {
-    event.target.blur();
```

Comment:
It would be wiser to just wrap it in a setTimeout.
`setTimeout(function(){
    event.target.blur()
});`
This way, the blur behavior remains and the "onchange" event fires.